app/views/meals/restrictions/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for(:title, "Meals Settings: Dietary Restrictions") %>
+<p>
+  Please enter the dietary restriction by how it is described when a meal 
+  contains it, and when there is an abscence of it, as both forms are used. 
+  We have given common restrictions.
```

Comment:
Can you remove `We have given common restrictions.` please? I don't think it is necessary and it may be confusing over time who "we" refers to